app/pages/patientdata/patientdata.js

```diff
@@ -526,6 +512,61 @@ export let PatientData = React.createClass({
       datetimeLocation: datetime
     });
   },
+
+  componentWillMount: function() {
+    this.doFetching(this.props);
+    this.doProcessing(this.props);
```

Comment:
I recall yesterday we took it out and there was a specific case for why it needed to be present on componentWillMount.